src/models/v3/asyncapi.ts

```diff
@@ -4,16 +4,22 @@ import { Info } from "./info";
 
 import { Mixin } from '../utils';
 import { ExtensionsMixin } from './mixins/extensions';
+import { ServersInterface } from "models/servers";
+import { Servers } from "./servers";
 
 export class AsyncAPIDocument 
   extends Mixin(BaseModel, ExtensionsMixin) 
   implements AsyncAPIDocumentInterface {
-		
+
   version(): string {
     return this.json("asyncapi");
   }
 
   info(): Info {
     return new Info(this.json("info"));
   }
+
+  servers(): ServersInterface {
+    return new Servers(this._json.servers);
+  }
```

Comment:
You are not creating a valid `Servers` instance. This class accepts `Server` instance arrays, not an array of normal objects:

```suggestion
  servers(): ServersInterface {
    return new Servers(
      Object.entries(this._json.servers).map(([serverName, server]) => new Server(serverName, server))
    );
  }
```

You have to import `Server` class.